README.md

```diff
@@ -109,6 +109,14 @@ commit your own `.semgrepignore`.
 Note that `.semgrepignore` is picked up only by the action,
 and will not be honored when running `semgrep` manually.
 
+### Audit Mode
+
+You can set `--audit-on event_name` to always report success,
+even if some findings were found.
+For example, in GitHub Actions, if you set the option `auditOn: push`,
+scans will run but not fail on push events.
+You can set multiple event names with this option.
```

Comment:
What event types are possible or supported? If these are GitHub specific, may we link to their docs?